src/baseclient.cc

```
@@ -2049,4 +2050,374 @@ UploadPartCopyResponse BaseClient::UploadPartCopy(UploadPartCopyArgs args) {
+    return RemoveObject(args);
+  });
+}
+
+std::future<RemoveObjectsResponse> BaseClient::RemoveObjectsAsync(
+    RemoveObjectsApiArgs args) {
+  return std::async(std::launch::async, [this, args = std::move(args)]() {
+    return BaseClient::RemoveObjects(args);
+  });
+}
+
+std::future<SelectObjectContentResponse> BaseClient::SelectObjectContentAsync(
```

Cppcheck (2.21.0)

[style] The function 'SelectObjectContentAsync' is never used.

(unusedFunction)